pyiron/vasp/base.py

```diff
@@ -2225,51 +2230,29 @@ def __init__(self, input_file_name=None, table_name="kpoints"):
             val_only=True,
             comment_char="!",
         )
-        self._trace = None
-
-    def _set_trace(self, trace):
-        """
-        Sets high symmetry points names of k-points trace (line mode only)
-
-        Args:
-            trace (list): new trace
-        """
-        self._trace = trace
+        self._path_name = None
+        self._n_path = None
 
-    def _get_trace(self):
-        """
-        Returns high symmetry points name of k-points trace (Line mode only)
-
-        Returns:
-            list: trace values
-        """
-        return self._trace
-
-    def set(self, method=None, size_of_mesh=None, shift=None, n_trace=None, trace_coord=None):
+    def set(self, method=None, size_of_mesh=None, shift=None, n_path=None, path=None):
```

Comment:
I think we need to rename this function in the `Kpoints` class since it overwrites the `set` function of the base class with different input parameters. This is why codeacy tests complain. We should probably call it `set_kpoints_file` or something. Sorry, I should have done this much earlier but I never got to it!